content/projects/terra/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Terra
+description: "This page provides the details of SODA DashBoard"
```

Comment:
This is the Terra project folder. 
You can add content from the SODA API, Controller, Dock projects here.

Reply:
There was description mentioned only in `Framework` and no user guides, developer guides for this so should i add that only?